guides/user/kubernetes-v2/rollout-strategies/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: single
+title:  "Rollout Strategies"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.14" %}
+
+{% include toc %}
+
+This guide describes how to take advantage of the
+[Kubernetes V2](/setup/install/providers/kubernetes-v2) provider's first-class support
+for common rollout strategies, including dark, highlander, and red/black rollouts.
+
+The implementation of these rollout strategies currently leverages Spinnaker's existing
+[traffic management strategy](/guides/user/kubernetes-v2/traffic-management/) and so is
+[valid for ReplicaSets only](/guides/user/kubernetes-v2/traffic-management/#you-must-use-replica-sets).
+
+## Rollout Strategy Options
```

Comment:
[Here](https://www.spinnaker.io/concepts/#deployment-strategies) and elsewhere we call them "deployment strategies." I think we should be consistent, if possible, but I guess that would mean a change in Deck too?

Reply:
Curious for @ethanfrogers' opinion on this, but @ezimanyi and I were afraid "deployment" was too loaded a term in the Kubernetes community since it refers to a specific Kubernetes kind (a kind you can't even use with these strategies, to boot!). But if we don't think that's a huge problem I'm all for consistency and happy to update Deck's wording as well.

Reply:
Ah. Interesting. 

One possibility is that "deployment strategy" is a term unto itself, separate from, and unconfused with, "deployment." But that might not be the case for users immersed in K8s. 

I defer to y'all, Maggie and Ethan and Eric. Consistency is good, but not if it adds confusion.

Reply:
+1 to Maggie's concern about confusion with the "Deployment" term in Kubernetes...though maybe "deployment strategy" is a common enough word it stands to itself?  I'd be fine with either...or we could jump right to "Traffic Management" to better encompass dark deploy.

Reply:
I agree that `Deployment` is overloaded and could be confusing. I like Rollout Strategies though, I have a _slight_ concern that it could be confused with a Deployment's `strategy` thus making the Deployment/ReplicaSet thing even _more_ confusing.

I think I prefer Traffic Management _most_ but we can always fallback to that if we find that we get a lot of questions about what Rollout Strategy means.

Reply:
My only concern with using Traffic Management is it seems like that's only one dimension of the rollout (the other being the creation/destruction of workloads), but yeah let's definitely keep an ear to the ground and record any confusion with the terminology. Going to merge this as-is for now and happy to make any future updates going forward based on feedback!